src/main/java/com/FaceCNN/faceRec/Dto/Response/FolderResponse.java

```diff
@@ -20,13 +17,8 @@ public static FolderResponse fromFolder(Folder folder) {
     }
 
 
-    public static String Sanitize(String folderPath){
-
-        if(folderPath == null || folderPath.isEmpty()) {
-            return FolderResponse.DEFAULT_FOLDER_NAME;
-        }
-
-        return folderPath.substring(folderPath.lastIndexOf("/")).replaceAll("/", "");
+    public static String Sanitize(String property){
```

Comment:
Se só for usar o método nesse record, deixar ele privado.
Nome do método camelCase.